Dockerfile

```diff
@@ -1,6 +1,27 @@
-FROM alpine:3.7
-MAINTAINER Nic Cope <n+docker@rk0n.org>
+FROM node as node
+ADD frontend/ .
+RUN npm install && npm run build
+
+FROM golang as golang
+WORKDIR /go/src/github.com/negz/kuberos/
+ENV GOOS="linux" GOARCH="amd64" CGO_ENABLED=0
+
+ADD . .
+COPY --from=node dist/ dist/frontend
+COPY --from=node index.html dist/frontend/
 
-RUN apk update && apk add ca-certificates
+RUN go get -u github.com/Masterminds/glide && \
+  go get -u github.com/rakyll/statik && \
+  glide install
 
-COPY "dist/kuberos" /
\ No newline at end of file
+RUN cd statik && go generate && cd ..
+RUN go build -o /kuberos ./cmd/kuberos
+
+FROM alpine:3.7 as ca-certificates
+RUN apk --no-cache add ca-certificates
+
+FROM scratch
```

Comment:
I'm not super familiar with using the `scratch` image. What are the advantages vs using `alpine`? I do find it useful to sometimes be able to `apk add` inside the Docker container to debug/triage things.

Reply:
`FROM scratch` results in an empty root filesystem, it works for statically linked binaries. The advantage is that the image is even smaller. ~8MB with scratch and ~16MB with alpine.  Yes, you would loose the ability to enter the container for troubleshooting.

I can set this change aside.